log.txt

```diff
@@ -0,0 +1,199 @@
+1202308651 Projects/Superalgos/Schemas/Docs-Tutorials/welcome-to-superalgos-013-tutorial-step-how-are-things-going.json                                  (Luis Fernando Molina 2021-01-08 21:03:04 +0100   1) {
+1202308651 Projects/Superalgos/Schemas/Docs-Tutorials/welcome-to-superalgos-013-tutorial-step-how-are-things-going.json                                  (Luis Fernando Molina 2021-01-08 21:03:04 +0100   2)     "tutorial": "Welcome to Superalgos!",
```

Comment:
Is this file meant to be submitted?

Reply:
> Is this file meant to be submitted?

I don't know.

Reply:
There doesn`t seem to be a path and i can`t find an existing log.txt file. You added it. Have you spoken with the docs Team or julian-molina that you will add this file or did it happen accidentally. If it happened accidentally could you remove it from this Pull request so the pull request can get merged?
Greetings :+1:

Reply:
I figured it out. I'll fix it soon.

Reply:
Done.

Reply:
Nice, i will look through the Pull request. This may take a time as it are some files.